src/sentry/conf/server.py

```diff
@@ -1907,6 +1909,7 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
     (
         ("member:admin", "Read, write, and admin access to organization members."),
         ("member:write", "Read and write access to organization members."),
+        ("member:invite", "Member invite access to organization members."),
         ("member:read", "Read access to organization members."),
```

Comment:
nit: Could we order the member scopes in this PR so it goes invite, read, write, admin?

Reply:
sorry should be other way around: admin, write, read, invite

Reply:
🐱